core/src/avm1/globals/bevel_filter.rs

```diff
@@ -226,7 +225,7 @@ impl<'gc> BevelFilter<'gc> {
         value: Option<&Value<'gc>>,
     ) -> Result<(), Error<'gc>> {
         if let Some(value) = value {
-            let strength = ((value.coerce_to_f64(activation)? * 256.0) as u16).clamp(0, 0xFF00);
+            let strength = value.coerce_to_f64(activation)?.clamp(0.0, 255.0);
```

Comment:
Actually low precision is needed; Take for example:
```as
var f = new flash.filters.BevelFilter();
f.strength = 1 / 257;
trace(f.strength); // 0
```
Whereas now it would trace some non-zero number.

Reply:
That should probably be added as a test.

Reply:
Darn ok. Yes I agree that should be in a test!